src/content/examples/CustomScrollbars/index.jsx

```diff
@@ -46,8 +47,8 @@ const Scrollbar4 = () => {
     );
 };
 
-const Scrollbars = ({children}) => (
-    <Scrollable>
+const Scrollbars = ({children, ...props}) => (
+    <Scrollable {...props}>
```

Comment:
I don't see you passing anything other than children anywhere

Reply:
yeah I passed the new prop (`cssVarsOnTracks`) in the last example but then decided to remove it. I can remove the spread also